python/sdk/client/models/model_endpoint_alert.py

```diff
@@ -48,19 +43,14 @@ class ModelEndpointAlert(object):
         'alert_conditions': 'alert_conditions'
     }
 
-    def __init__(self, model_id=None, model_endpoint_id=None, environment_name=None, team_name=None, alert_conditions=None, _configuration=None):  # noqa: E501
```

Comment:
Do you know what is the `_configuration` used for?

Reply:
Other than being used by api_client, looking through old generated codes seems to suggest that it is `_configuration ` is not used. In some cases where they are called, the function also does not exist in the default `Configuration()`. For example:
https://github.com/swagger-api/swagger-codegen/blob/621a7330ee7940623af34f935eee9fda6992a300/samples/client/petstore/python/petstore_api/models/animal.py#L82

which seems to suggest there is no real need for the `_configuration` object.

Reply:
Got it, thanks!